src/db.js

```diff
@@ -15,18 +15,13 @@ const {
   POSTGRES_DB,
 } = process.env;
 
-/*
-  All credentials setted to default values (exsept password - it is exapmle)
-  replace if needed with your own
-*/
-
 const sequelize = new Sequelize({
   database: POSTGRES_DB || 'postgres',
   username: POSTGRES_USER || 'postgres',
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '123123',
 });
```

Comment:
You are using the '||' operator to provide default values for your environment variables. This is not a good practice as it can lead to unexpected behavior if the environment variables are not set correctly. It would be better to validate the environment variables and throw an error if they are not set correctly.